.github/workflows/pr.yaml

```
@@ -15,16 +15,16 @@ jobs:
       - checks
       - cpp-build-test
     secrets: inherit
-    uses: rapidsai/shared-workflows/.github/workflows/pr-builder.yaml@branch-24.04
+    uses: rapidsai/shared-workflows/.github/workflows/pr-builder.yaml@branch-24.08
```

Please use 24.10, since that's the current development branch. (Applies to all files in `.github/workflows/`)
```suggestion
    uses: rapidsai/shared-workflows/.github/workflows/pr-builder.yaml@branch-24.10
```